scanner/updater/version/RELEASE_VERSION

```diff
@@ -5,4 +5,4 @@
 # This process ensures that each version's vulnerability data is accurately captured and maintained in accordance with its respective version.
 4.3.x-173-g6bbb2e07dc
 4.3.0
-4.3.x-nightly-20240105
\ No newline at end of file
+4.3.x-nightly-202401054.4.0
```

Comment:
@daynewlee Seems like we need the new line in this file. Do you know if there is a reason we don't have one at the moment?

Reply:
This will be an automate step in the release:
https://redhat-internal.slack.com/archives/C03D78P4RFF/p1706104001305419?thread_ts=1705444427.650449&cid=C03D78P4RFF

Reply:
I am not sure. It should be a automate step in the release

Reply:
I believe the issue is because the current version of the file is missing the newline. Maybe for this release, we should manually fix this

Reply:
I manually fixed it for this PR. I do expect future PRs won't have this problem

Reply:
Thank you!